llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -3266,18 +3269,22 @@ bool SIInstrInfo::FoldImmediate(MachineInstr &UseMI, MachineInstr &DefMI,
       if (pseudoToMCOpcode(NewOpc) == -1)
         return false;
 
-      // We need to swap operands 0 and 1 since madmk constant is at operand 1.
+      // V_FMAMK_F16_t16 takes VGPR_32_Lo128 operands, so the rewrite
```

Comment:
Isn't this part in a separate PR?

Reply:
No, #66202 handles the `V_FMAAK_F16_t16` case above.